packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -95,12 +89,12 @@ const ImageGridFallbackTopStory = css`
   width: ${fullWidthColumnsMaxScaleable};
 
   @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
-    width: ${halfWidthColumnsMaxScaleable};
+    width: calc(${halfWidthColumnsMaxScaleable} - ${GEL_SPACING});
```

Comment:
I think it would be better to declare this as a variable at the top of this file and then pull that in here and on line 97 as well.

Reply:
Added a `gridFallbackImageWidth` variable to hold the calculation